File: Java/Desktop/NewLab8/src/by/bntu/fitr/povt/cng/model/entity/Lights.java
package by.bntu.fitr.povt.cng.model.entity;

import by.bntu.fitr.povt.cng.model.entity.abstracts.Toy;

public class Lights extends Toy {
    private int lifeTime;
    private int usualTemp;
    private int colorfulTemperature;

    public Lights() {

    }

    public Lights(double price, double weight, double size, int lifeTime, int usualTemp, int colorfulTemperature) {
        super(price, weight, size);
        this.lifeTime = lifeTime;
        this.usualTemp = usualTemp;
        this.colorfulTemperature = colorfulTemperature;
    }



    public int getLifeTime() {
        return lifeTime;
    }

    public void setLifeTime(int lifeTime) {
        this.lifeTime = lifeTime;
    }

    public int getUsualTemp() {
        return usualTemp;
    }

    public void setUsualTemp(int usualTemp) {
        this.usualTemp = usualTemp;
    }

    public int getColorfulTemperature() {
        return colorfulTemperature;
    }

    public void setColorfulTemperature(int colorfulTemperature) {
        this.colorfulTemperature = colorfulTemperature;
    }

    @Override
    public String toString() {
        return new StringBuilder().append("\nLights "+ super.toString()).toString(); //+
//                "\n\tLife time: " + lifeTime +
//                "\n\tUsual temp: " + usualTemp +
//                "\n\tColorful temperature: " + colorfulTemperature).toString();
    }
}

File: Csharp/Lab8/Lab8/Lab8/cng/Model/Entity/Lights.cs
using System.Text;
using Lab8.cng.Model.Entity.Abstracts;

namespace Lab8.cng.Model.Entity
{
    public class Lights : Toy
    {
        private int _lifeTime;
        private int _usualTemp;
        private int _colorfulTemperature;

        public Lights() {

        }

        public Lights(double price, double weight, double size, int lifeTime, int usualTemp, int colorfulTemperature) : base(price, weight, size){
            
            this._lifeTime = lifeTime;
            this._usualTemp = usualTemp;
            this._colorfulTemperature = colorfulTemperature;
        }



        public int GetLifeTime() {
            return _lifeTime;
        }

        public void SetLifeTime(int lifeTime) {
            this._lifeTime = lifeTime;
        }

        public int GetUsualTemp() {
            return _usualTemp;
        }

        public void SetUsualTemp(int usualTemp) {
            this._usualTemp = usualTemp;
        }

        public int GetColorfulTemperature() {
            return _colorfulTemperature;
        }

        public void SetColorfulTemperature(int colorfulTemperature) {
            this._colorfulTemperature = colorfulTemperature;
        }

        
        public override string ToString() {
            return new StringBuilder().Append("\nLights "+ base.ToString() +
                                              "\n\tLife time: " + _lifeTime +
                                              "\n\tUsual temp: " + _usualTemp +
                                              "\n\tColorful temperature: " + _colorfulTemperature).ToString();
        }
    }
}